installer/cape2.sh

```diff
@@ -604,43 +741,44 @@ function install_yara() {
 }
 
 function install_mongo(){
-    echo "[+] Installing MongoDB"
-    # Mongo >=5 requires CPU AVX instruction support https://www.mongodb.com/docs/manual/administration/production-notes/#x86_64
-    if grep -q ' avx ' /proc/cpuinfo; then
-        MONGO_VERSION="6.0"
-    else
-        echo "[-] Mongo >= 5 is not supported"
-        MONGO_VERSION="4.4"
-    fi
-
-    sudo curl -fsSL "https://www.mongodb.org/static/pgp/server-${MONGO_VERSION}.asc" | sudo gpg --dearmor -o /etc/apt/keyrings/mongo.gpg --yes
-    echo "deb [signed-by=/etc/apt/keyrings/mongo.gpg arch=amd64] https://repo.mongodb.org/apt/ubuntu $(lsb_release -cs)/mongodb-org/${MONGO_VERSION} multiverse" > /etc/apt/sources.list.d/mongodb.list
+	if ( "$mongo_enable" -ge 1 ); then
+		echo "[+] Installing MongoDB"
+		# Mongo >=5 requires CPU AVX instruction support https://www.mongodb.com/docs/manual/administration/production-notes/#x86_64
+		if grep -q ' avx ' /proc/cpuinfo; then
+			MONGO_VERSION="6.0"
+		else
+			echo "[-] Mongo >= 5 is not supported"
+			MONGO_VERSION="4.4"
+		fi
+
+		sudo curl -fsSL "https://www.mongodb.org/static/pgp/server-${MONGO_VERSION}.asc" | sudo gpg --dearmor -o /etc/apt/keyrings/mongo.gpg --yes
+		echo "deb [signed-by=/etc/apt/keyrings/mongo.gpg arch=amd64] https://repo.mongodb.org/apt/ubuntu $(lsb_release -cs)/mongodb-org/${MONGO_VERSION} multiverse" > /etc/apt/sources.list.d/mongodb.list
```

Comment:
I'd also recommend something more like
`echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/6.0 multiverse" | sudo tee /etc/apt/sources.list.d/mongodb-org-${MONGO_VERSION}.list`
I would imagine you would get a permission denied for most systems. 
https://www.mongodb.com/docs/manual/tutorial/install-mongodb-on-ubuntu/

Reply:
Using sudo won't really do anything meaningful here given this all needs to run as root any ways. Really no good reason to call sudo from this script unless you need to run it as something other than root.

Reply:
Yeah, the sudo wasn't the point with that command. My pipeline was complaining about not having permissions to run that line, but it was due to something else. This one is resolved. Sorry.